Microsoft.Toolkit.Uwp.UI/Behaviors/FocusBehavior.cs

```diff
@@ -0,0 +1,174 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Markup;
+
+namespace Microsoft.Toolkit.Uwp.UI.Behaviors
+{
+#pragma warning disable SA1402 // File may only contain a single type
+    /// <summary>
+    /// This behavior sets the focus on the first control of <see cref="Targets"/> which accepts it.
+    /// The focus will be set following the <see cref="Targets"/> order. The first control being ready
+    /// and accepting the focus will receive it.
+    /// The focus can be set to another control with a higher priority if it loads before <see cref="FocusEngagementTimeout"/>.
```

Comment:
`FocusEngagementTimeout` is private, yet you are mentioning it in this public comment. Does it make sense to make FocusEngagementTimeout a public DP that can be changed?

Reply:
I've moved the value to a DP.